wiki/fabrication/fabrication_considerations_for_3D_printing.md

```diff
@@ -0,0 +1,158 @@
+
+Printing Filament Properties:
+- PLA filament is a stiff but brittle 3D printing material.
+- Well suited for prototyping
+- Best for low-stress applications. Should not be the major load bearing of a mechanical structure.
+- Best 3D printer material for beginners due to ease of printing and minimal warp.
+
+Technologies: FDM, SLA, SLS
+
+PLA material properties for printing:
+- Strength: High | Flexibility: Low | Durability: Medium
+- Difficulty to use: Low/li>
+- Print temperature: 180°C – 230°C
+- Print bed temperature: 20°C – 60°C (but not needed)
+- Shrinkage/warping: Minimal
+- Soluble: No
+
+
+
+### ABS
+Acrylonitrile Butadiene Styrene or ABS for short, is created from Acrylonitrile, Butadiene and Styrene polymers. It is a material commonly used in personal or household 3D printing, which is done using primarily FDM. If you are used to printing with PLA, you’ll probably find ABS a little trickier to print with.
+
+ABS Filament: What are the Proper Settings?
+It can be quite some task for a newcomer to find the right settings. No spool is alike, and the specifications may vary. Make sure you cover these bases:
+
+- Close the printer: Before printing, make sure your 3D printer is either closed or that the ambient temperature of your environment isn’t too hot or too cold.
+- Use a heated bed: Set the temperature to 110 degrees Celcius. Also, think about using Kapton (blue) on your glass build plate. If you still have problems with the first layer adhesion of your ABS filament, you should use a raft.
+- Experiment with the temperature: Set the nozzle temperature to 230 degrees and work your way from there.
+- If you see a lot of plastic strings between separate parts and the printer oozes material, lower the temperature by 5 degrees.
+- If the filament doesn’t stick, the parts aren’t sturdy or the surfaces have a rough feel, increase the temperature by 5 degrees.
+
+
+ABS 3D Printing Filament Properties:
+
+- ABS filament is strong, ductile material with wear resistance and heat tolerance.
+- Common 3D prints with ABS are Interlocking parts like gears, parts exposed to UV and heat like a car cup holder, or prototyping.
+- Wide selection of methods for excellent post-processing.
+
+Technologies: FDM, Binder Jetting, SLA, PolyJetting
+
+ABS properties:
+- Strength: High | Flexibility: Medium | Durability: High
+- Difficulty to use: Medium
+- Print temperature: 210°C – 250°C
+- Print bed temperature: 80°C – 110°C
+- Shrinkage/ warping: Considerable
+- Soluble: In esters, ketones, and acetone.
+
+   [create_docker_image]: <https://www.mirantis.com/blog/how-do-i-create-a-new-docker-image-for-my-application/>
+   [install_nvidia_driver]: <https://github.com/heethesh/Computer-Vision-and-Deep-Learning-Setup>
+
+#### Furthur Reading:
+1. Design considerations for 3D printing.
+<https://www.3dhubs.com/knowledge-base/key-design-considerations-3d-printing>
```

Comment:
Wrong format. Should be [title](link)

Reply:
Correct the format for the entire '_Furthur Reading_' section.